packages/client/mutations/connections/getUserTasksConn.ts

```diff
@@ -1,6 +1,6 @@
 import {ConnectionHandler, RecordProxy} from 'relay-runtime'
 
-const getUserTasksConn = (viewer: RecordProxy, userIds: string[] | null, teamIds: string[] | null) =>
+const getUserTasksConn = (viewer: RecordProxy, userIds?: string[] | null, teamIds?: string[] | null) =>
```

Comment:
this is tricky. i'm not sure if relay treats null as being equivalent to undefined. i don't think it does, which means it may not find the right connection